src/ai/defaultai.cc

```diff
@@ -1555,7 +1567,7 @@ void DefaultAI::update_buildable_field(BuildableField& field) {
 	resource_count_now = false;
 	// Testing in first 10 seconds or if last testing was more then 60 sec ago
 	if (field.last_resources_check_time < Time(10000) ||
-	    field.last_resources_check_time > gametime + Duration(60 * 1000)) {
+	    gametime > field.last_resources_check_time + Duration(60 * 1000)) {
```

Comment:
this was simply a bug preventing the fileds to be updated resulting in quarries being built without stones in vicinity